.github/CODEOWNERS

```diff
@@ -8,6 +8,7 @@ packages/theme/** @shopify/advanced-edits @shopify/app-inner-loop
 .changeset/* @shopify/advanced-edits @shopify/app-inner-loop
 .github/CODEOWNERS @shopify/advanced-edits @shopify/app-inner-loop
 packages/cli/oclif.manifest.json @shopify/advanced-edits @shopify/app-inner-loop
-packages/cli-kit/src/{private,public}/themes/* @shopify/advanced-edits @shopify/app-inner-loop
+packages/cli-kit/src/private/themes/* @shopify/advanced-edits @shopify/app-inner-loop
```

Comment:
nit: in general I'd sort this file lexicographically, just in the name of cleanliness and helping folks the right place if they need to add more entries